xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -780,12 +780,10 @@ absl::Status IrEmitterUnnested::EmitCublasLtMatmulThunkF8(
       BufferAllocation::Slice b_scale,
       GetAllocationSliceForHlo(instr->operand(a_scale_index + 1)));
 #if GOOGLE_CUDA
-  TF_ASSIGN_OR_RETURN(
-      BufferAllocation::Slice c_scale,
-      GetAllocationSliceForHlo(instr->operand(a_scale_index + 2)));
+  BufferAllocation::Slice c_scale;
```

Comment:
Move this out of the #if and delete line 788. Maybe add a comment as well.

Reply:
Oh, I must have forgotten to push the new change. Re-pushing.